common/libretro.c

```diff
@@ -682,6 +682,9 @@ static void update_variables(bool startup)
          log_cb(RETRO_LOG_INFO, "Got size: %u x %u.\n", width, height);
 
       initial_resolution_set = true;
+   } else {
```

Comment:
I think this is a bit worrying. I know plenty of targets like the console versions for which a software resolution of 640x480 will be way too slow. You might want to pick way lower defaults, 320x200/320x240 or something.